supplementary_style_guide/style_guidelines/release-notes.adoc

```diff
@@ -74,6 +74,35 @@ If you refer to those tickets without including a link, inform the user. See the
 
 “Some referenced tickets are not linked. This means that the ticket is not accessible without Red Hat credentials."
 
+[[release-note-asciidoc-format]]
+== Release note formatting in AsciiDoc
+To display individual release notes outside of the structure of the table of contents, treat each release note as a description list item. This format is also compatible with the DITA/AEM migration. For additional formatting examples, you can inspect the link:https://redhat-documentation.github.io/supplementary-style-guide/#release-notes[Release notes] file in the `supplementary-style-guide` repository.
+
+.Release note AsciiDoc basic formatting template
+----
+Release note heading::
+This is the main release note text.
++
+Add another paragraph if necessary.
++
+link:https://issues.redhat.com/browse/TICKET-REFERENCE[TICKET-REFERENCE]
+----
+
+For the DITA conversion to work correctly, the list must remain uninterrupted. Attach all lines after the first to the description with a plus sign. Do not use open blocks (`--`) to separate elements within the description. If you need to introduce a list inside a release note text, add an empty line before attaching the reference line with a plus sign to preserve continuation.
```

Comment:
Is "attach" the right word here? Maybe it should say something like this?

```suggestion
For the DITA conversion to work correctly, the list must remain uninterrupted. When creating the description list, put a plus sign before each item in the list. Do not use open blocks (`--`) to separate elements within the description. If you need to introduce a list inside a release note text, add an empty line before attaching the reference line with a plus sign to preserve continuation.
```
I'm also not sure what the "reference line" is. I think what you're saying is that you must have a blank line before you add more lines to the release note, but I'm not sure. I'm a little lost.

Reply:
I think we'll need to discuss this on Monday.